index.html

```diff
@@ -9,11 +9,14 @@
   <script type="text/babel">
     const root = document.getElementById("root");
     function App() {
-      const [counter, modifier] = React.useState(0);
+      const [counter, setCounter] = React.useState(0);
+      const onClick = () => {
+        setCounter(counter + 1);
```

Comment:
The component is rendered again, this means re created again, a new const is created with the new value of the 'counter'